govcd/catalog_test.go

```diff
@@ -84,3 +96,179 @@ func (vcd *TestVCD) Test_DeleteCatalog(check *C) {
 	check.Assert(catalog, Equals, Catalog{})
 
 }
+
+func (vcd *TestVCD) Test_UploadOvf(check *C) {
+	checkUploadOvf(vcd, check, getCurrentPath()+OvaPath, TestCreateCatalog, "item1")
+}
+
+func (vcd *TestVCD) Test_UploadOvf_chuncked(check *C) {
+	checkUploadOvf(vcd, check, getCurrentPath()+OvaChunkedPath, TestCreateCatalog+"2", "item2")
+}
+
+func (vcd *TestVCD) Test_UploadOvf_progress_works(check *C) {
+	catalogName := TestCreateCatalog + "3"
+	itemName := "item3"
```

Comment:
It's test independence question. Test can't depend on other tests result (e.g. what id did with catalog in previous test).

Reply:
If more catalogs are needed, we should name them according to their purposes. 
In general, we should make tests that work with the material provided by users (i.e the catalog defined in the configuration file).
We should create new catalogs only when testing something relating to catalog creation or if we run an operation that requires two catalogs. Otherwise, we should always use the catalog given by users.

Reply:
Fixed